### PR TITLE
Add edits from ASM, Add Stephen Mack, and update contributions

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- Add introduction edits from Ashley M
- Add Stephen Mack as author for review
- Added review and edit contribution to everyone who reviewed and/or edited
- Updated title to have abbreviation `OpenPBTA`

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -366,7 +375,9 @@ authors:
   initials: AJW
   orcid: 0000-0002-0571-2889
   email: awaanders@luriechildrens.org
-  contributions: Supervison
+  contributions:
+  - Supervison
```
Comment (sjspielman) [inline content/metadata.yaml:379]:
```suggestion
  - Supervision
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -251,6 +255,7 @@ authors:
   contributions:
   - Investigation
   - Validation
+  - Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:258]:
```suggestion
  - Writing - Review and editing
```

File: content/metadata.yaml [hunk 3]
```diff
@@ -310,7 +315,9 @@ authors:
   initials: DRS
   orcid: 0000-0001-8193-1488
   email: drstewart@mail.nih.gov
-  contributions: Supervision
+  contributions:
+  - Supervision
+  - Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:320]:
```suggestion
  - Writing - Review and editing
```

File: content/metadata.yaml [hunk 4]
```diff
@@ -431,7 +442,9 @@ authors:
   initials: JK
   orcid: 0000-0001-6274-2841
   email: jung.kim2@nih.gov
-  contributions: Investigation
+  contributions:
+  - Investigation
+  - Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:447]:
```suggestion
  - Writing - Review and editing
```

File: content/metadata.yaml [hunk 5]
```diff
@@ -589,6 +602,7 @@ authors:
   - Visualization
   - Writing - Original draft
   - Supervision
+  - Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:605]:
```suggestion
  - Writing - Review and editing
```

File: content/metadata.yaml [hunk 6]
```diff
@@ -615,6 +629,21 @@ authors:
   - NIH U2C HL138346-03
   - NCI/NIH Contract No. 75N91019D00024, Task Order No. 75N91020F00003
   - Australian Government, Department of Education
+- name: Ashley S. Margol
+  initials: ASM
+  orcid: 0000-0002-3038-8005
+  email: amargol@chla.usc.edu
+  contributions: Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:636]:
```suggestion
  contributions: Writing - Review and editing
```

File: content/metadata.yaml [hunk 7]
```diff
@@ -615,6 +629,21 @@ authors:
...
+  affiliations:
+  - Division of Hematology and Oncology, Children’s Hospital Los Angeles
+  - Department of Pediatrics, Keck School of Medicine of University of Southern California
+- name: Stephen C. Mack
+  initials: SCM
+  orcid: 0000-0001-9620-4742
+  email: stephen.mack@stjude.org
+  contributions: Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:644]:
```suggestion
  contributions: Writing - Review and editing
```

File: content/metadata.yaml [hunk 8]
```diff
@@ -675,6 +704,7 @@ authors:
   - Software
   - Supervision
   - Writing – original draft
+  - Writing - review and editing
```
Comment (sjspielman) [inline content/metadata.yaml:707]:
```suggestion
  - Writing - Review and editing
```

**Review by jaclyn-taroni — APPROVED** [review]
Assuming you will take @sjspielman's suggestions, this looks good to go

**Review by sjspielman — APPROVED** [review]

**Pull request merged**